.travis.yml

```diff
@@ -0,0 +1,90 @@
+language: php
+
+notifications:
+    email:
+        recipients:
+            - dev@dmk-ebusiness.de
+        on_success: change
+        on_failure: always
+
+php:
+    - 5.6
+    - 7.0
+    - 7.1
+    - 7.2
+
+env:
+    - TYPO3_VERSION="^6.2.14"
+    - TYPO3_VERSION="^7.6.0"
+    - TYPO3_VERSION="^8.7.0"
+    - TYPO3_VERSION="^9.5.0"
+
+path:
+    - TEST_PATH = "tests"
+
+matrix:
+    exclude:
+        - php: 7.0
+          env: TYPO3_VERSION="^6.2.14"
+          path: TEST_PATH = "tests"
+        - php: 7.1
+          env: TYPO3_VERSION="^6.2.14"
+          path: TEST_PATH = "tests"
+        - php: 7.2
+          env: TYPO3_VERSION="^6.2.14"
+          path: TEST_PATH = "tests"
+        - php: 7.2
+          env: TYPO3_VERSION="^7.6.0"
+          path: TEST_PATH = "tests"
+        - php: 5.6
+          env: TYPO3_VERSION="^8.7.0"
+          path: TEST_PATH = "tests"
+        - php: 5.6
+          env: TYPO3_VERSION="^9.5.0"
+          path: TEST_PATH = "tests"
+        - php: 7.0
+          env: TYPO3_VERSION="^9.5.0"
+          path: TEST_PATH = "tests"
+        - php: 7.1
+          env: TYPO3_VERSION="^9.5.0"
+          path: TEST_PATH = "tests"
```

Comment:
We should add PHP Version 7.2 for TYPO3 9.5 to the test matrix.

Maybe we could consider 7.3 as well.